package.json

```diff
@@ -32,13 +86,13 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "dotenv": "^8.1.0",
-    "eslint": "^6.1.0",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint": "^5.12.1",
```

Comment:
Down a version?

Reply:
It makes this repo conform to the other repos.

Reply:
By this logic we update singular dependencies of all repositories at once across the entire org? 😄

Reply:
(I'm fine with it)

Reply:
It also doesn't work on my machine as configured, nor Jimmys if I remember a prior conversation correctly.

Reply:
👍